data/human/news.txt

```diff
@@ -546,11 +546,12 @@ news "hyperspace relay engineer"
 			`"`
 		word
 			"All of human space is linked into a single communications network by relay beacons. It's something that most people just take for granted."
-			"Hyperspace links don't just allow faster-than-light travel. They also allow near-instantaneous communication, if you have the right sort of relay on both ends."
+			"Hyperspace links don't just allow faster-than-light travel. If you have the right sort of relay on both ends, they also allow rapid communication."
 			"The hyperspace communications network is a marvel of public infrastructure, but most people just take it for granted."
 			"Hyperspace relays are so integral to society that even pirates don't dare to touch them. They'd easily become the number one enemy of the Republic if they did."
 			"I'm surprised at how many people on Earth are still using the old IPv4 protocol. None of the off-Earth websites are reachable for them."
 			"Have you ever set up a TCP/IP connection from Bourne to a server on Farpoint? I did and it is a little slow, but it works!"
+			"It might take hours to get a message a few systems away going through the hyperspace relays, but it sure beats taking multiple days to send a ship or multiple centuries to send a message a light speed."
```

Comment:
```suggestion
			"It might take hours to get a message a few systems away going through the hyperspace relays, but it sure beats taking multiple days to send a ship or multiple centuries to send a message at light speed."
```